app/src/main/java/fr/free/nrw/commons/upload/UploadPresenter.java

```diff
@@ -32,6 +34,9 @@
     private UploadContract.View view = DUMMY;
 
     private CompositeDisposable compositeDisposable;
+    public static final String COUNTER_OF_NO_LOCATION
+        = "how_many_consecutive_uploads_have_no_coordinates";
```

Comment:
@misaochan Do we have a convention for these store key values? I think there was an issue about that, but can't find it.

Reply:
Not as far as I'm aware, @neslihanturan any idea?

Reply:
@Ayan-10 The feature works great, and the code looks fine. Could you please just rename `COUNTER_OF_NO_LOCATION` to `COUNTER_OF_CONSECUTIVE_UPLOADS_WITHOUT_COORDINATES` (sorry a bit long but explicit) and `how_many_consecutive_uploads_have_no_coordinates` to `number_of_consecutive_uploads_without_coordinates`, if it sounds good to you? Thanks!

Reply:
That looks more accurate. Sure, I will change it.